airflow/operators/empty.py

```diff
@@ -29,6 +29,7 @@ class EmptyOperator(BaseOperator):
 
     ui_color = '#e8f7e4'
     inherits_from_empty_operator = True
+    custom_operator_name = 'Empty'
```

Comment:
Only remaining question is if we want this (or should we leave it as `EmptyOperator`)

Reply:
Ah, yes, I did that when updating the tests to prove I wasn't just accidentally getting `task_type`. I'll revert that and use a specific class that's just in the tests.